cmake/SFMLConfig.cmake.in

```diff
@@ -132,7 +132,7 @@ if (EXISTS "${targets_config_file}")
             string(TOUPPER "${component}" UPPER_COMPONENT)
             if (TARGET SFML::${component})
                 set(SFML_${UPPER_COMPONENT}_FOUND TRUE)
-            else()
+            elseif(SFML_FIND_REQUIRED_${component})
                 set(FIND_SFML_ERROR "Found SFML but requested component '${component}' is missing in the config defined in ${SFML_DIR}.")
                 set(SFML_${UPPER_COMPONENT}_FOUND FALSE)
                 set(SFML_FOUND FALSE)
```

Comment:
https://cmake.org/cmake/help/latest/command/find_package.html#:~:text=Additional%20optional%20components%20may%20be%20listed%20after%20OPTIONAL_COMPONENTS.%20If%20these%20cannot%20be%20satisfied%2C%20the%20package%20overall%20can%20still%20be%20considered%20found%2C%20as%20long%20as%20all%20required%20components%20are%20satisfied.

This suggestion would break documented behavior.

Reply:
Addressed in 5c29fb127415f3ee289a27ddfafa195a5fb55036 (force push with amended commit)